### DISCUSSION TITLE
Can't hide bottom tab bar in the stack screen

### DISCUSSION THREAD

**Discussion by Pavel-Ananich** [react-navigation/react-navigation#10094] [Q&A]
### Current behavior

I put PTNavigator into BottomTabNavigator
![Screenshot 2021-11-02 at 16 24 38](https://user-images.githubusercontent.com/86955927/139855427-3c277149-0d8a-48e9-9a15-9462812a2e32.png)

And I need to hide bottom tab bar on a specific screen in stack navigator:
![Screenshot 2021-11-02 at 16 26 27](https://user-images.githubusercontent.com/86955927/139855751-4ba9e9db-40c7-4558-9b2f-95382deabb81.png)

But nothing changed. Bottom tab bar is still visible on every screen.

### Expected behavior

Bottom tab bar must be hidden. I didn't find in docs how to hide it.

### Reproduction

https://snack.expo.dev/P5Dk-t-_n

### Platform

- [X] Android
- [X] iOS
- [ ] Web
- [ ] Windows
- [ ] MacOS

### Packages

- [ ] @react-navigation/bottom-tabs
- [ ] @react-navigation/drawer
- [ ] @react-navigation/material-bottom-tabs
- [ ] @react-navigation/material-top-tabs
- [X] @react-navigation/stack
- [ ] @react-navigation/native-stack

### Environment

| package                                | version |
| -------------------------------------- | ------- |
| @react-navigation/native               ^6.0.6
| @react-navigation/bottom-tabs          ^6.0.9
| @react-navigation/drawer               don't use
| @react-navigation/material-bottom-tabs  don't use
| @react-navigation/material-top-tabs    don't use
| @react-navigation/stack                don't use
| @react-navigation/native-stack         ^6.1.0
| react-native-safe-area-context         ^3.3.2
| react-native-screens                   ^3.7.2
| react-native-gesture-handler           ^1.10.3
| react-native-reanimated                don't use
| react-native-tab-view                  don't use
| react-native-pager-view                don't use
| react-native                           0.66.1
| expo                                   don't use
| node                                   17.0.1
| npm                      8.1.0

**Comment by raajnadar** [comment]
You can read this docs https://reactnavigation.org/docs/hiding-tabbar-in-screens

**Reply by Pavel-Ananich** [reply]
I read it, but if you look into the problem, you will see, that it doesn't help, because I need tab bar on some stacks screens, but on some I don't need

**Reply by raajnadar** [reply]
If you read carefully there is a solution.

I found that you are sending `tabBarStyle: { display: 'none' }` to the wrong navigator, you need to send that option to bottom tab bar. Because it is a bottom tab bar option.

**Reply by Pavel-Ananich** [reply]
Ok, if I send It to bottom tab navigator, tab bar will be hidden on every screen, but I need to hide it only on some stack screens. Here is my code, you can see, why this article can't help me:

![Screenshot 2021-11-04 at 13 25 15](https://user-images.githubusercontent.com/86955927/140297897-e0a784ff-8537-4852-8c6d-11e412fff683.png)
![Screenshot 2021-11-04 at 13 25 35](https://user-images.githubusercontent.com/86955927/140297944-9953faaf-ab28-4413-b108-dbb670a7dc79.png)
![Screenshot 2021-11-04 at 13 25 53](https://user-images.githubusercontent.com/86955927/140297992-5e81fd0c-f2fb-4790-9875-8138d1a9b012.png)

**Reply by raajnadar** [reply]
The stack screen in which you don't want the bottom tab should be outside the tab navigator, there is no other option. Restructuring the navigation is the only solution.

**Comment by userhv** [comment]
Still unresolved? I also need this behavior.
